qgispluginci/release.py

```diff
@@ -452,7 +452,7 @@ def upload_plugin_to_osgeo(
             f"Plugin path: {archive}"
         )
         logger.error(err_msg)
```

Comment:
@Guts Even without the debug option, the error is displayed with `logger.error()` so whatever the debug level, this error is shown.